guides/user/managed-delivery/resource-status/index.md

```diff
@@ -128,18 +128,32 @@ When you resume management the status will change to [Resumed](/guides/user/mana
   image_path="./resource-status-resumed.png"
 %}
 
-When a resource has a Resumed status, it means management was just resumed after being temporarily paused and Spinnaker hasn't checked for drift from the declarative configuration yet.
+When a resource has this status, it means management was just resumed after being temporarily paused and Spinnaker hasn't checked for drift from the declarative configuration yet.
 
 If a drift is detected, the status will change to [Diff](/guides/user/managed-delivery/resource-status/#diff). If no drift is detected, the status will change to [Happy](/guides/user/managed-delivery/resource-status/#unhappy).
 
+### Currently unresolvable
+{%
+  include
+  figure
+  image_path="./resource-status-currently-unresolvable.png"
+%}
+
+When a resource has this status, it means something Spinnaker requires to properly manage the resource is either not ready yet or temporarily experiencing disruption. Automatic action won't be taken while in this status, but it doesn't necessarily mean something is broken. Some examples of things that might lead to a resource entering this status:
+
+  - A downstream system (like a container registry or cloud provider) temporarily rate limiting or experiencing availability issues
```

Comment:
Love the examples here, I think it's super helpful! I wonder if we can add some resources here that can help users to figure out what is wrong, like troubleshooting links? (I don't really know if we have something like this)

Reply:
yeah i share that desire — sadly it's pretty specific to the error and many times will involve company-internal systems. I think we _can_ make an impact on this though by adding helpful guidance and recommendations to the error messages we add to history events where useful!